src/makeCalculator.js

```diff
@@ -1,10 +1,41 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(a) {
+      this.result += a;
+
+      return this.result;
+    },
+    subtract(a) {
+      this.result -= a;
+
+      return this.result;
+    },
+    multiply(a) {
+      this.result *= a;
+
+      return this.result;
```

Comment:
[CODE KNOWLEDGE] - The `multiply` method should not return the result directly. In order to support chaining, it should return the calculator object itself.